src/task.py

```diff
@@ -540,10 +567,33 @@ def is_gridcell_unique(self, df):
         return df_counts.index.is_unique
 
     def calc(self):
+        print(self.df_adhoc)
```

Comment:
We should comment this so a future developer knows why we have to print and not remove them. And add a card to refactor so we don't have to do this.